src/parser/validator.rs

```diff
@@ -530,6 +530,52 @@ impl Conflicts {
             conf
         })
     }
+
+    fn gather_overrides(&mut self, cmd: &Command, matcher: &ArgMatcher, arg_id: &Id) -> Vec<Id> {
+        debug!("Conflicts::gather_overrides: arg={:?}", arg_id);
```

Comment:
Could we have an intermediate commit that breaks out these functions, duplicating the other behavior, and then the commit after changes it to the new behavior?  It'd make it a lot easier to see what the behavior difference is.